src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -686,8 +686,12 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
   _getParsedTextTracks(): Array<PKTextTrack> {
     const captionsTextTrackLabels = [this._config.captionsTextTrack1Label, this._config.captionsTextTrack2Label];
     const captionsTextTrackLanguageCodes = [this._config.captionsTextTrack1LanguageCode, this._config.captionsTextTrack2LanguageCode];
+    const captionAvailable = !!Array.from(this._videoElement.textTracks).find(
+      track => track.kind === 'captions' && ((track.activeCues && track.activeCues.length > 0) || track.mode === 'showing')
```

Comment:
it should be cues in general and not just activeCues
Why do we need showing state? maybe because you do not check just cues? I think it is wrong cause native video player may have this option set and then mode will be `showing`